google/cloud/storage/benchmarks/async_throughput_benchmark.cc

```diff
@@ -320,7 +320,7 @@ auto MakeAsyncClients(Configuration const& cfg,
 
 auto MakeClient(ClientConfig const& cc, int background_threads) {
   if (cc.transport == "GRPC") {
-    return google::cloud::storage_experimental::DefaultGrpcClient(
+    return gcs::MakeGrpcClient(
```

Comment:
comment: I would have made the library code changes in one PR, then the build changes in another PR. All good, though.

Reply:
Yea good point. I had started with older work Carlos had done back in April which included everything unfortunately.